app/views/shared/_header.html.haml

```diff
@@ -1,4 +1,10 @@
 - unless Rails.env.development?
   :javascript
     window.onbeforeunload = function() { return true };
-= image_tag logo_image, style: 'width:20%; margin: auto; display: block;', class: 'right-align'
+.row
+  .col.s12
+    - if Rails.application.config.settings.use_company_logo
+      = image_tag Rails.application.config.settings.logo.company_logo, style: 'width: 20%; display: inline; vertical-align: middle'
```

Comment:
misschien, om generiek te zijn, willen we die images gewoon 20% zo groot maken, ipv hier die width zetten.

```bash
convert image.png -resize 20%  image.png
```

Reply:
(en dan ook voor die andere images)

Reply:
Nee die width geeft aan dat het plaatje 20% breedte van de pagina moet bevatten, niet dat het plaatje op 20% breedte van haar originele grootte moet worden weergegeven.

Dus als je de pagina groter of kleiner maakt dan schalen de logos mee in grootte. Je wilt dus juist dat ze groot (genoeg) zijn. Ook wil je ze eigenlijk 2 keer zo groot als dat je ze daadwerkelijk toont zodat ze er ook altijd goed op retina uitzien.